dbt/adapters/athena/impl.py

```diff
@@ -538,29 +536,6 @@ def _s3_path_exists(self, s3_bucket: str, s3_prefix: str) -> bool:
         response = s3_client.list_objects_v2(Bucket=s3_bucket, Prefix=s3_prefix)
         return True if "Contents" in response else False
 
-    def _join_catalog_table_owners(self, table: agate.Table, manifest: Manifest) -> agate.Table:
```

Comment:
I'm happy to see this removed, because it simplify the code...and at the end "dbt" owner is still available in the manifest.